include/zephyr/bluetooth/audio/bap.h

```diff
@@ -529,6 +529,17 @@ struct bt_bap_stream_ops {
 	void (*released)(struct bt_bap_stream *stream);
 #endif /* CONFIG_BT_BAP_UNICAST */
 
+	/**
+	 * @brief Stream started callback
+	 *
+	 * Almost-started callback is called whenever an Audio Stream has been started
+	 * in terms of the CIS Establish and sending Receiver Start Ready to the remote
+	 * device, but the Streaming notification is still awaited.
+	 *
+	 * @param stream Stream object that has been started.
+	 */
+	void (*almost_started)(struct bt_bap_stream *stream);
+
```

Comment:
We actually used to have `connected` and `disconnected` here for the ISO channels, to let the application know when the ISO channels state changed, which is of course tied to the stream state, but not 100% (as your point here is). 

Maybe we should bring those back, as also suggested by https://github.com/zephyrproject-rtos/zephyr/issues/60216

Reply:
Yes, that would be the best solution.

Reply:
@mkasenberg is https://github.com/zephyrproject-rtos/zephyr/issues/60216 something you'd want to take on as a task? 

Else I can get started on that next week.

Reply:
I haven't started working on this yet, so I'll leave it to you then.

Reply:
Alright :)